emmygozi/Fastchop
# #160651922 Setup database connection

#### What does this PR do?
Setup database connection
#### Description of Task to be completed?
- Setup database locally with scripts that run on custom api
- Setup database to persist application data
- Sets database environment variable with dotenv
- Sets database environment variable on travis for test purpose
- Sets database environment variable on heroku
#### How should this be manually tested?
- Download dotenv and specify environment variables of your local system in ".env" file
- Create database in pgAdmin
- Run "yarn start:dev" or "npm run start:dev"
- Send a GET / request by visiting http://localhost:8000/api/v1/tablesyeah
   a) This creates all the required tables "users", "menu" and "orders" table
   b) It then select all (select *) from orders table which is a table that has foreign key userid and menuid
   c) This makes sure that "user" and "menu" table has been created.
- Expected response:
>{
     "message":"Retrieved empty table orders",
      "orders":[]
  }
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/160651922

#### Any background context you want to add?
Our database connection checks for current environment; "test", "development" or 
"production".
#### Screenshots
![image](https://user-images.githubusercontent.com/32358797/45916660-a7231700-be61-11e8-8747-a632aead699d.png)
